.github/workflows/unit-tests.yml

```diff
@@ -154,22 +154,36 @@ jobs:
       with:
         environment-file: ci/deps/${{ matrix.env_file }}
 
+    - name: Set sanitizer flags
+      run: |
+        echo "CFLAGS=$CFLAGS -fno-sanitize-recover=all" >> "$GITHUB_ENV"
+    - name: echo flags
+      run: |
+        printf '%s\n' "CFLAGS"
     - name: Build Pandas
       id: build
       uses: ./.github/actions/build_pandas
+      env:
+        CFLAGS: "$CFLAGS"
```

Comment:
IIRC, this doesn't set CFLAGS inside of the action. It just makes ``env.CFLAGS`` available.

Why don't you try setting CFLAGs in action.yml if sanitize = true?

Reply:
Ah great idea - much cleaner